arch/arm/src/stm32/hardware/stm32f20xxx_pinmap_legacy.h

```diff
@@ -0,0 +1,685 @@
+/****************************************************************************
+ * arch/arm/src/stm32/hardware/stm32f20xxx_pinmap_legacy.h
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+#ifndef __ARCH_ARM_SRC_STM32_HARDWARE_STM32F20XXX_PINMAP_LEGACY.H
+#define __ARCH_ARM_SRC_STM32_HARDWARE_STM32F20XXX_PINMAP_LEGACY.H
```

Comment:
```suggestion
#ifndef __ARCH_ARM_SRC_STM32_HARDWARE_STM32F20XXX_PINMAP_LEGACY_H
```
here and in other places

Reply:
@raiden00pl Thanks Got these in last push